infor-design/enterprise-wc
# 1596 - Remove locales from bundles

**Explain the details for making this change. What existing problem does the pull request solve?**

Changes all the locale files from ts/js to .json. By doing these they have the effect of removing them from the bundles in down stream projects.

**Related github/jira issue (required)**:
Fixes #1596 

**Steps necessary to review your pull request (required)**:
 - pull and `npm i` and `npm run start`
 - test some locale pages like:
 - http://localhost:4300/ids-date-picker/example.html -> changing locale in the .. menu to de-De should fetch the locale json and update the picker
 - http://localhost:4300/ids-locale/example.html -> changing locale in the .. menu should update the test on that page
 - run `npm run build:dist` and `npm run build:dist:prod`
 - copy build/dist of choice over to https://github.com/infor-design/enterprise-wc-examples/tree/main/angular-ids-wc / node modules and make sure that still works
 - check out  https://github.com/infor-design/enterprise-wc-examples/tree/main/angular-ids-wc and run `npm i` then `npm run build:stats` notice the problem with the locales in the bundle
 - copy build/dist of choice over to https://github.com/infor-design/enterprise-wc-examples/tree/main/angular-ids-wc /(https://github.com/infor-design/enterprise-ng-quickstartnode modules and run `npm run build:stats` should now not have the stuff in the bundle..

 **Screen Shot Results**:
 
<img width="1446" alt="Screenshot 2023-11-17 at 2 20 52 PM" src="https://github.com/infor-design/enterprise-wc/assets/814283/63cdc390-a05e-4668-b485-938f838f0ecf">

**Included in this Pull Request**:
